### PR TITLE
server GUI: fix initial volume button value

### LINKED ISSUE #1370: Server Gui volume bug (gitreports.com)
Submitter: patrick
Email: p.dupuis@live.com
Fedora 21 Linux - 3.19.1-201.fc21.x86_64
Supercollider 3.7alpha 
commit 4bcab22

When I create a server window with s.makeGui, the volume slider in the server window is initially set to 0 db regardless of the IDE status bar volume setting. It would be nice if the server window took into account the status bar volume when it is created. 

The two volumes begin to match when either of them is changed after the server window's creation.

### PR DESCRIPTION
**Pull request by gusano**
Fixes #1370

### CODE REVIEW

File: SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc [hunk 1]
```diff
@@ -277,7 +280,7 @@
 
 			volumeNum = gui.numberBox.new(w, Rect(0, 0, 28, 18))
 				.font_(font)
-				.value_(0.0)
+				.value_(currentVolume.clip(-90, 6))
```
Comment (muellmusik) [inline SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc]: Correct me if I'm wrong, but I don't think you need to clip here. The Volume object itself clips, and can be set to have different ranges than -90 or 6, so not good to enforce the defaults in the GUI, esp as they may contradict the actual state.

Comment (gusano) [inline SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc]: You're right.

**Comment by gusano** [comment]
@muellmusik just pushed the change.

**Comment by muellmusik** [comment]
Otherwise this looks good to me, and quite minor. Any objection to merging?

**Comment by muellmusik** [comment]
Build problem is not related. Here's the issue:
```ifmac brew update
remote: fatal: packfile /data/repositories/a/nw/a0/76/48/206084/network.git/objects/pack/pack-b96a27a4be5c6b39c120147daf4d4f34e14f39fe.pack cannot be accessed
remote: aborting due to possible repository corruption on the remote side.
fatal: protocol error: bad pack header
Error: Failure while executing: git pull -q origin refs/heads/master:refs/remotes/origin/master
The command "ifmac brew update" failed and exited with 1 during 
```
BTW, how do you rerun a travis build if it fails spuriously?

**Comment by gusano** [comment]
> BTW, how do you rerun a travis build if it fails spuriously?

There's an arrow icon on the right, but you might need to be logged in into travis to see it.


travis

I just restarted it..

**Comment by muellmusik** [comment]
> There's an arrow icon on the right, but you might need to be logged in into travis to see it.

Ah is that the same as Github or do you need a different account?

**Comment by gusano** [comment]
Your Github account.

**Pull request merged**